app/views/catalog/_statement_on_language_in_description.html.erb

```diff
@@ -1,2 +1,2 @@
 <dt>Statement on language in description</dt>
-<dd>Princeton University Library aims to describe library materials in a manner that is respectful to the individuals and communities who create, use, and are represented in the collections we manage. <a href="https://library.princeton.edu/about/languagestatement" target="_blank" title="Opens in a new tab">Read more...</a></dd>
+<dd>Princeton University Library aims to describe library materials in a manner that is respectful to the individuals and communities who create, use, and are represented in the collections we manage. <a href="https://library.princeton.edu/about/languagestatement" target="_blank" title="Opens in a new tab" aria-label="Full Statement on Language in Description">Read more...</a></dd>
```

Comment:
same here: Can we update the aria-label to: `aria-label="Read more about full Statement on Language in Description"`